src/TwigYard/Component/Application.php

```diff
@@ -59,40 +70,65 @@ public function run()
         $server->listen();
     }
 
+    /**
+     * @return ContainerBuilder
+     */
+    public function createContainer()
+    {
+        $containerBuilder = new ContainerBuilder();
```

Comment:
Could we please move this to a new `ContainerFactory` component. That would allow as to pass in `ContainerBuilder` and `YamlFileLoader` by DI

Reply:
Fixed